packages/components/src/components/dropdown/dropdown.stories.ts

```diff
@@ -1,17 +1,23 @@
 import '../../solid-components';
 import { html } from 'lit';
 import { storybookDefaults, storybookHelpers, storybookTemplate } from '../../../scripts/storybook/helper';
-import { userEvent } from '@storybook/test';
-import { waitUntil } from '@open-wc/testing-helpers';
 import { withActions } from '@storybook/addon-actions/decorator';
 
 const { argTypes, parameters } = storybookDefaults('sd-dropdown');
 const { generateTemplate } = storybookTemplate('sd-dropdown');
 const { overrideArgs } = storybookHelpers('sd-dropdown');
 
+/**
+ * Displays a list of actions or options in a panel when activated.
+ *
+ * **Related templates:**
+ * - [Dropdown with navigation items](?path=/docs/templates-dropdown-with-navigation-items--docs)
+ */
+
 export default {
   title: 'Components/sd-dropdown',
   component: 'sd-dropdown',
+  tags: ['!dev'],
   args: overrideArgs([
     {
       type: 'slot',
```

Comment:
Utilize sd-navigation-item as trigger.

See updated https://www.figma.com/design/YDktJcseQIIQbsuCpoKS4V/Component-Docs?node-id=2357-10912&t=UpPGX0jehyNFfgNq-11

Reply:
And display slot without background color / like on Figma.

Reply:
Updated.